systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -198,7 +198,8 @@ public class Environment {
     private static final String ST_KAFKA_VERSION_DEFAULT = TestKafkaVersion.getDefaultSupportedKafkaVersion();
     private static final String ST_CLIENTS_KAFKA_VERSION_DEFAULT = "3.9.0";
     public static final String TEST_CLIENTS_VERSION_DEFAULT = "0.10.0";
-    public static final String ST_FILE_PLUGIN_URL_DEFAULT = "https://repo1.maven.org/maven2/org/apache/kafka/connect-file/" + ST_KAFKA_VERSION_DEFAULT + "/connect-file-" + ST_KAFKA_VERSION_DEFAULT + ".jar";
+    //public static final String ST_FILE_PLUGIN_URL_DEFAULT = "https://repo1.maven.org/maven2/org/apache/kafka/connect-file/" + ST_KAFKA_VERSION_DEFAULT + "/connect-file-" + ST_KAFKA_VERSION_DEFAULT + ".jar";
```

Comment:
can we remove this comment?

Reply:
Once 4.0 is released, you need to uncomment the original line and delete the new line. That is why I keep this.